File: xyident/baekjoon/4단계/2562___.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

let index = 0;
let max = 0;

for(let i = 0; i < 9; i++){
    if(input[i] > max){
        index = i;
        max = input[i];
    }
}

console.log(max);
console.log(index + 1);

File: rolemadelen/baekjoon/4단계/2562.rs
use std::io::{stdin};

fn main() {
    let mut max = -1;
    let mut pos = -1;
    (0..9).for_each(|idx| {
        let mut input = String::new();
        stdin().read_line(&mut input).unwrap();
        let v: i32 = input.trim().parse().unwrap();

        if v > max {
            pos = idx + 1;
            max = v;
        }
    });

    println!("{}\n{}", max, pos);
}

